### PR TITLE
Implement upload for xlsx file

### PR DESCRIPTION
**Pull request by busykoala**
To access the reports we want to have the possibility to choose a filelisting block in the installation where the file is uploaded.

### CODE REVIEW

File: ftw/linkchecker/command/checking_links.py [hunk 1]
```diff
@@ -322,7 +321,24 @@ def send_mail_with_excel_report_attached(email_address, plone_site_obj,
...
+        file_listing_block = portal.unrestrictedTraverse(path=filelistingblock_url.encode('utf-8'))
+    except Exception as e:
+        # TODO: log exception (this is most likely be raised if
+        # the path to a file listing block is wrong.
+        raise
+
+    xlsx_file.seek(0)
+    data = xlsx_file.read()
+
+    file = plone.api.content.create(container=file_listing_block, type='File', title=file_name, file=data)
```
Comment (maethu) [inline ftw/linkchecker/command/checking_links.py]: Don't override pythons `file` here. You could simply use `file_`, oder `file_content_type`

File: ftw/linkchecker/command/checking_links.py [hunk 2]
```diff
@@ -322,7 +321,24 @@ def send_mail_with_excel_report_attached(email_address, plone_site_obj,
...
-        email_subject, email_message, email_address, xlsx_file)
+        email_subject, email_message, email_address, xlsx_file, file_name)
+
+
+def upload_report_to_filelistingblock(filelistingblock_url, xlsx_file, file_name):
+    portal = api.portal.get()
+    try:
+        file_listing_block = portal.unrestrictedTraverse(path=filelistingblock_url.encode('utf-8'))
+    except Exception as e:
+        # TODO: log exception (this is most likely be raised if
```
Comment (maethu) [inline ftw/linkchecker/command/checking_links.py]: Yes please, I would like to see if the upload fails for some reason,

**Review by maethu — CHANGES_REQUESTED** [review]
Looks already pretty good, just two minor things.

**Comment by tinagerber** [comment]
Okay, I've made the desired changes.

**Review by maethu — APPROVED** [review]

**Pull request merged**